web/src/features/fireWeather/components/TimeOfInterestPicker.tsx

```diff
@@ -23,11 +22,11 @@ const TimeOfInterestPicker = (props: Props) => {
   return (
     <TextField
       data-testid="time-of-interest-picker"
+      className={classes.datePicker}
       label="Time of Interest (PST-08:00)"
       type="datetime-local"
-      value={formatDateInPST(props.timeOfInterest, 'YYYY-MM-DDTHH:mm')}
+      value={props.timeOfInterestISO.slice(0, 16)} // 'YYYY-MM-DDTHH:mm'
```

Comment:
what if we sliced this at (0, 13) instead? Then we could get rid of the minutes selection, which doesn't have any effect anyway

Reply:
Unfortunately, the datepicker component doesn't accept `YYYY-MM-DDTHH` format